stylesheets/commons/Brackets.less

```diff
@@ -631,6 +631,7 @@ div.brkts-popup-body-element-thumbs {
 				height: unset;
 				border: 0.0625rem solid #ffffff;
```

Comment:
Should this border still be here?

Reply:
i'll defer to you what you think is best

Reply:
I changed it to outline, made more sense